build/citations.py

```diff
@@ -2,6 +2,7 @@
 from hashlib import blake2b
 import pathlib
 import re
+import sys
```

Comment:
> Unused import?
Removed

> What is Center OS? http://www.centeros.com/ or https://www.centos.org/ ?
 Sorry. It is CentOS.

> Instead it defaults to whatever `locale.getpreferredencoding()` returns.

On the system we are using, it returns "ANSI_X3.4-1968"